opennms-container/minion/container-fs/entrypoint.sh

```diff
@@ -163,6 +163,14 @@ applyConfd() {
   fi
 }
 
+applyOpennmsPropertiesD() {
+  for filename in ${MINION_HOME}/etc/opennms.properties.d/*.properties; do
```

Comment:
Is it possible there are cases where this directory isn't created? Does it need to be checked for existence?

Reply:
The directory is always there because [instance-id.properties.toml](https://github.com/OpenNMS/opennms/blob/develop/opennms-container/minion/container-fs/confd/conf.d/instance-id.properties.toml) is always run.

Reply:
> Should we get rid of [instance-id.properties.toml](https://github.com/OpenNMS/opennms/blob/develop/opennms-container/minion/container-fs/confd/conf.d/instance-id.properties.toml) since it is just another system property ?

I do not know why `instance-id.properties` was introduced in the first place. I think @indigo423 did introduce it. If we delete it then we have to make sure that OpenNMS still gets that setting.

Reply:
looks like it's dependency of some other external projects, we can let it go through for now rather than trying to rework it